client/cloud_credentials.go

```diff
@@ -160,13 +146,27 @@ func (client *ApiClient) CloudCredentialsList() ([]Credentials, error) {
 	return credentials, nil
 }
 
-func (client *ApiClient) CredentialsCreate(request CredentialCreatePayload) (Credentials, error) {
-	organizationId, err := client.OrganizationId()
+func (client *ApiClient) CredentialsCreate(request any) (Credentials, error) {
+	// Convert to map to set organizationId
+	requestMap, err := convertToMap(request)
```

Comment:
golang magic.
convert to a map of string[any].
check if projectId is defined in the map.
if it is - do nothing. just convert back to original struct.
if it does not exist calculate the organizationId, add it, and convert back to struct.

This is done for both create and update operations.